MOCC/src/configurations/NewPyTango/HealthMonitorDS.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, jsonify
+from flasgger import Swagger
+
+
+app = Flask(__name__)
+app.config['SWAGGER'] = {
+    'title': 'Health Monitor API',
+    'uiversion': 2
+}
+Swagger(app)
```

Comment:
do we want something like https://github.com/rochacbruno/flasgger/blob/master/examples/restful.py
it uses flask_restful and adds resources like todos to the api.

Do we want do something similar?
Something like:

```
from flask import Flask
from flask_restful import Api, Resource, abort, reqparse

from flasgger import Swagger, swag_from

app = Flask(__name__)
api = Api(app)
app.config['SWAGGER'] = {
    'title': 'Flasgger RESTful',
    'uiversion': 2
}
swag = Swagger(app)

class HealthMonitor(Resource):
    def get(self):
        return (?)


api.add_resource(HealthMonitor, '/todos')

if __name__ == '__main__':
    app.run(debug=True)
```